server/src/main/java/org/opensearch/search/profile/query/QueryProfiler.java

```diff
@@ -104,4 +109,11 @@ public CollectorResult getCollector() {
         return collector.getCollectorTree();
     }
 
+    /**
+     * Return the top level breakdown for plugins to use
+     */
+    public ContextualProfileBreakdown getProfileBreakdown(Query query) {
```

Comment:
Could we expose `AbstractProfileBreakdown` instead, any particular reason to expose wider type?
```suggestion
    public AbstractProfileBreakdown getProfileBreakdown(Query query) {
```